tests/AppAny.Quartz.EntityFrameworkCore.Migrations.Tests/IntegrationTests.cs

```diff
@@ -3,12 +3,16 @@
 
 namespace AppAny.Quartz.EntityFrameworkCore.Migrations.Tests
 {
+  using System;
```

Comment:
It is pretty common namespace and inside this file is single namespace, so no reason to put this using here to separate scopes